From b240ba3c94452f31349161716f76882f8938f41d Mon Sep 17 00:00:00 2001
From: Yuya Hamamachi <yuya.hamamachi.sx@renesas.com>
Date: Wed, 12 Jan 2022 15:58:18 +0900
Subject: [PATCH 2/3] starterkit: linux: rom_writer: Change flash_writer path

This patch changes flash_writer path in linux script.
As a result, Windows and Linux script can use same flash_writer binary.

Signed-off-by: Yuya Hamamachi <yuya.hamamachi.sx@renesas.com>
---
 starterkit/linux/script/rom_writer | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/starterkit/linux/script/rom_writer b/starterkit/linux/script/rom_writer
index c4ad9a3..471fe0f 100755
--- a/starterkit/linux/script/rom_writer
+++ b/starterkit/linux/script/rom_writer
@@ -16,7 +16,7 @@ set tty  [lindex $argv 4]
 
 proc mot {} {
     global top
-    return $top/linux/script/flash_writer/AArch32_output/AArch32_Flash_writer_SCIF_DUMMY_CERT_E6300400_ULCB.mot
+    return $top/AArch32_Flash_writer_SCIF_DUMMY_CERT_E6300400_ULCB.mot
 }
 
 proc cpld_mode {} {
-- 
2.31.1

